### PR TITLE
Change go-bindata version for system-probe

### PR DESCRIPTION
**Pull request by leeavital**
### What does this PR do?

Right now, inv -e system-probe.build will fetch an older version of
go-bindata.

inv -e deps will fetch a newer version of go-bindata (and pin that
version to a tag configured in bootstrap.json.)

The two versions have a similar API, so we should standardize on one
version to use.

### Motivation

@DataDog/networks 
@DataDog/container-integrations 

### Additional Notes

Anything else we should know when reviewing?

### Describe your test plan

Make sure the system-probe starts.

### CODE REVIEW

**Review by p-lambert — APPROVED** [review]
Thanks for fixing this! Just curious, how did we end up using different versions?
Shouldn't `go get -u github.com/jteeuwen/go-bindata/...` always download the latest one? Last time this package was updated was 5 years ago, which makes things even more confusing to me 🤔

**Comment by leeavital** [comment]
The agent is using shuLhan/go-bindata  which is newer.  

https://github.com/DataDog/datadog-agent/blob/master/bootstrap.json#L15

File: pkg/ebpf/tcp_queue_length.go [hunk 1]
```diff
@@ -29,7 +29,7 @@ type TCPQueueLengthTracer struct {
 }
 
 func NewTCPQueueLengthTracer() (*TCPQueueLengthTracer, error) {
-	source_raw, err := bytecode.Asset("tcp-queue-length-kern.c")
+	source_raw, err := bytecode.Asset("pkg/ebpf/c/tracer-ebpf-debug.o")
```
Comment (stb2132) [inline pkg/ebpf/tcp_queue_length.go:32]: Just wondering why this changed from tcp-queue-length-kern.c. Do we not use this code anymore, or was it incorporated with the tracer-ebpf-debug code?

Comment (leeavital) [reply pkg/ebpf/tcp_queue_length.go:32]: Yikes, this should be the path to tcq-queue-length-kern.c -- nice catch!

**Review by mx-psi — APPROVED** [review]

**Pull request merged**